sdk/cosmos/changelog/README.md

```diff
@@ -1,4 +1,8 @@
 ## Changelog
+### 3.7.0
+- Downgrade guava dependency to 25.0-jre from 27.0.1-jre
+- Fix FeedResponse bug (see https://github.com/Azure/azure-sdk-for-java/pull/8085)
```

Comment:
isn't this `FeedOptions` bug rather than `FeedResponse` bug?

Reply:
Yeah it is :D 
I will update it, thanks for catching it.

Reply:
Done